.github/workflows/bump-versions.yaml

```diff
@@ -17,6 +17,7 @@ jobs:
       - uses: actions/checkout@v4
       - run: |
           sed -i 's/version=.*/version=${{ github.event.inputs.version }}/' gradle.properties
+          cd php-custom-rules-plugin/maven && mvn versions:set -DgenerateBackupPoms=false -DnewVersion=${{ github.event.inputs.version }}
```

Comment:
Please test that it's working. You can trigger the workflow from this branch.

Reply:
It works! https://github.com/SonarSource/sonar-php/pull/1324
(I'm bumping the custom rules version to 3.40-SNAPSHOT too, missed that)